test/shape_attributes.cc

```diff
@@ -80,6 +80,66 @@ TEST(ShapeAttributes, test_shape_attributes_included) {
   }
 }
 
+TEST(ShapeAttributes, test_shape_attributes_duplicated_point) {
+  tyr::actor_t actor(conf);
+
+  auto result_json = actor.trace_attributes(
```

Comment:
This test passes on master as well...

Reply:
@PureW the test will pass, but is `ubsan` triggered?

Reply:
I'm not sure how to test UB without ubsan, I'll enable it on CI

Reply:
i would be fine to just undo the fix in triplegbuilder, turn on ubsan locally, show that the sanitizer finds the issue via the unit test and post it to this PR. we shouldnt change the behavior of CI in this PR, iirc when we had that enabled we either got randomly OOM killed or the build took forever

Reply:
In my opinion, CI should test valhalla for UB, it would help to avoid UB issues in future, It's not the only issue found with ubsan.